optaplanner-quarkus-integration/optaplanner-quarkus-benchmark/runtime/src/main/java/org/optaplanner/benchmark/quarkus/OptaPlannerBenchmarkBeanProvider.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.benchmark.quarkus;
+
+import java.io.File;
+import java.util.Collections;
+
+import javax.enterprise.inject.Produces;
+import javax.inject.Singleton;
+
+import org.optaplanner.benchmark.api.PlannerBenchmarkFactory;
+import org.optaplanner.benchmark.config.PlannerBenchmarkConfig;
+import org.optaplanner.benchmark.config.ProblemBenchmarksConfig;
+import org.optaplanner.benchmark.config.SolverBenchmarkConfig;
+import org.optaplanner.core.config.solver.SolverConfig;
+import org.optaplanner.core.config.solver.termination.TerminationConfig;
+
+import io.quarkus.arc.DefaultBean;
+
+public class OptaPlannerBenchmarkBeanProvider {
+
+    @DefaultBean
+    @Singleton
+    @Produces
+    PlannerBenchmarkFactory benchmarkFactory(OptaPlannerBenchmarkRuntimeConfig benchmarkRuntimeConfig,
+            SolverConfig solverConfig) {
+        ProblemBenchmarksConfig problemBenchmarksConfig = new ProblemBenchmarksConfig();
+        problemBenchmarksConfig.setWriteOutputSolutionEnabled(false);
```

Comment:
Why do we hardcode this for users? Generally, I like this to be `true`.

Reply:
I think we used to do it that way, but it caused issues.

- For serious use cases, with serious datasets, writing the output causes disk space issues.
Users should run the benchmarks a lot. If each benchmark takes a significant diskspace size (which is not noticeable in small use cases), it becomes annoying and it can even slow down the IDE IIRC.

- Also not, all FileIO's support writing the solution to disk. And it's non-trivial to implement that.
- Using in memory objects to benchmar (so no FileIO), means they can't be written to the output anyway.

I recommend keeping it to false, even though I understand the sentiment of just having all old solutions available too.

Reply:
Even if we keep it to `false`, why are we hardcoding it at all?
The user did not specify it, so we should just stick to the default, not override it.